vars/piperPipelineStageInit.groovy

```diff
@@ -172,6 +176,11 @@ void call(Map parameters = [:]) {
             .withMandatoryProperty('buildTool')
             .use()
 
+        if (config.transportRequestReqIDFromGit) {
+            echo "Retrieving transportRequestID from git commit history"
+            transportRequestReqIDFromGit(script: script)
```

Comment:
To get the right request id out of the git commit tree can be tricky. See [transportRequestReqIDFromGit](https://www.project-piper.io/steps/transportRequestReqIDFromGit).
The documentation has to clarify which options are supported (gitFrom, gitTo?) and provide samples!
The UI has to reflect these options accordingly